### DISCUSSION TITLE
Github Student developer pack delay

### DISCUSSION THREAD

**Discussion by Sidd280625** [community/community#137379] [GitHub Education] [answered]
### Hi everyone,

I have not received my GitHub student developer pack, I applied for it 5 days ago, When I checked it said if I don't receive it for over 5 days raise a discussion on the GitHub community, So I am raising a discussion on Github, I hope you see this request and help me

**Comment by davevad93** [comment, marked as answer]
Hi @Sidd280624 ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @Sidd280624 was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks; 

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Reply by Sidd280625** [reply]
thanks i received it but it was deep in my emails

**Reply by davevad93** [reply]
You're welcome @Sidd280624 👍
